docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -159,6 +164,23 @@ Additionally, the listener for `pbPreviewReloaded` has been removed.
 |`fbFormBuilderLoaded`|`ez-form-builder-loaded`|
 |`fbFormBuilderUnloaded`|`ez-form-builder-unloaded`
 
+#### Code cleanup
+
+Selected deprecated functions have been dropped.
+
+|Dropped code|Use instead|
+|------------|-----------|
+|`forcedLanguage`|`allowedLanguages`|
```

Comment:
I don't think we should mention this change in the documentation. This property is from smaller components from UDW. Personally I think we would be overly precise mentioning this change here. :)